docs/discover/document-explorer.asciidoc

```diff
@@ -22,65 +43,83 @@ click *Move left* or *Move right*.
 In the pop-up, drag the column names to their new order.
 
 * To resize a column, drag the right edge of the column header until the column is the width that you want.
-
++
 Column widths are stored with a saved search.  When you visualize saved searches on dashboards, the saved search appears the same as in **Discover**.
 
+
 [float]
 [[document-explorer-row-height]]
-=== Adjust row height
+==== Adjust row height
 
 To set the row height to 1 or more lines, or automatically
 adjust the height to fit the contents, click the row height icon
 image:images/row-height-icon.png[icon to open the Row height pop-up].
 
 [role="screenshot"]
-image::images/document-explorer-row-height.png[Row height settings for the document table, width="75%"]
-
+image::images/document-explorer-row-height.png[Row height settings for the document table, width="50%"]
 
 [float]
 [[document-explorer-sort-data]]
-=== Sort data
+==== Sort the fields
 
-Sort your data by one or more fields, in ascending or descending order.
+Sort the data by one or more fields, in ascending or descending order.
 The default sort is based on the time field, from new to old.
 
 To add or remove a sort on a single field, click the
 column header, and then select the sort order.
 
 To sort by multiple fields:
 
-. In the document table, click the *field sorted* option.
+. Click the *field sorted* option.
 +
 [role="screenshot"]
-image::images/document-explorer-sort-data.png[Pop-up in document table for sorting columns, width="75%"]
+image::images/document-explorer-sort-data.png[Pop-up in document table for sorting columns, width="50%"]
 
-. To add more fields to the sort, select from the dropdown menu.
+. To add fields to the sort, select their names from the dropdown menu.
 +
 By default, columns are sorted in the order they are added.
 +
 [role="screenshot"]
-image::images/document-explorer-multi-field.png[Multi field sort in the document table, width="75%"]
+image::images/document-explorer-multi-field.png[Multi field sort in the document table, width="50%"]
 
 . To change the sort order, select a field in the pop-up, and then drag it to the new location.
 
 
 [float]
 [[document-explorer-compare-data]]
-=== Compare data
+==== Filter the documents
 
-Narrow your results to a subset of documents to easily
-compare the data of interest.
+Narrow your results to a subset of documents so you're comparing the just data of interest.
 
 . Select the documents you want to compare.
 
 . Click the *documents selected* option, and then select *Show selected documents only*.
 +
 [role="screenshot"]
-image::images/document-explorer-compare-data.png[Compare data in the document table, width="75%"]
+image::images/document-explorer-compare-data.png[Compare data in the document table, width="50%"]
+
+[float]
+[[document-explorer-configure-table]]
+==== Set rows per page
+
+By default, the document table displays 100 rows per page. To change this number,
+click *Rows per page*.
```

Comment:
We now also have a new Advanced Setting `discover:sampleRowsPerPage` to customize the default Rows per page number.